src/dotnet/Fable.Tools/State.fs

```diff
@@ -260,9 +261,22 @@ let compile (com: Compiler) (project: Project) (fileName: string) =
             Fable2Babel.Compiler.createFacade fileName lastFile
     else
         printfn "Compile %s" fileName
-        FSharp2Fable.Compiler.transformFile com project project.CheckedProject fileName
-        |> Fable2Babel.Compiler.transformFile com project
-    |> addLogs com
+        let usingFableCore =  (com :> ICompiler).Options.fableCore = "fable-core"
+        let nextCompiler = 
+            match usingFableCore with
+            | false -> com
+            | true -> 
+                let prevPlugins = (com :> ICompiler).Plugins
+                let prevOptions = (com :> ICompiler).Options
+                let fableCoreLocation = 
+                    let paketDirectory = project.PaketDirectory
+                    IO.Path.Combine(paketDirectory, "packages", "Fable.Core", "fable-core") 
+                    |> Parser.makePathRelative
+                let nextOptions = { prevOptions with fableCore = fableCoreLocation }
+                Compiler(nextOptions, prevPlugins)
+        FSharp2Fable.Compiler.transformFile nextCompiler project project.CheckedProject fileName
+        |> Fable2Babel.Compiler.transformFile nextCompiler project
+        |> addLogs nextCompiler
```

Comment:
If we move the indentation here it means we are not applying `addLogs` to the other `if` branches any more.